### DISCUSSION TITLE
Let the user select directories instead of files

### DISCUSSION THREAD

**Discussion by Jaaneek** [Jaaneek/useFilePicker#31] [Ideas]
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory

We would implement it as useDirectoryPicker
It won't work on all browsers

Your thoughts on this?

**Comment by MrKampla** [comment]
Actually, despite it being a non-standard property, webkitdirectory has a quite good support across major browsers (around 95%). It won't work on IE 11, but every other browser has been supporting it for many years.
![image](https://user-images.githubusercontent.com/46401284/142679027-f6280d21-2f41-486e-9738-777c8826f51f.png)

Also, I've recently encountered a package: https://www.npmjs.com/package/browser-fs-access 
that aims to bring a file system access API to browsers. This API is in a working draft state and not even official (https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API). It's maintained by Google Chrome labs, and they're also using webkitdirectory as a fallback, so it looks like a legit solution to me.

To sum up, we can easily implement a separate useDirectoryPicker hook that will be exported through the main file src/index.tsx just like existing useFilePicker. My question is, which props should useDirectoryPicker receive? Do You have any ideas?
